modules/deployments-comparing-deploymentconfigs.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * applications_and_projects/deployments/what-deployments-are.adoc
+
+[id='deployments-comparing-deploymentconfigs-{context}']
+= Comparing Deployments and DeploymentConfigs
+
+Because deployment configurations existed in {product-title} prior to
+deployments being added in Kubernetes 1.2, the latter object type naturally
+diverges slightly from the former. The long-term goal in {product-title} is to reach
+full feature parity in Deployments and switch to using them as a
+single object type that provides fine-grained management over applications.
+
+Deployments are supported to ensure upstream projects and examples that use the
```

Comment:
What if we just said "Both Deployments and DeploymentConfigs are supported in {product-title}." instead of this sentence? 

(Also I'm going to update everywhere that we say "deployment configuration" in this file to "DeploymentConfig", as I've been told that's how we should be writing it in the docs).

Reply:
+1 to both